app/index.html

```diff
@@ -13,6 +13,12 @@
     <link rel="stylesheet" href="{{rootURL}}assets/hack.css">
     <!--Import Google Icon Font-->
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" type="text/css">
+    <!-- Compiled and minified CSS -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/css/materialize.min.css">
```

Comment:
This here - https://github.com/coding-blocks/hacker-blocks/blob/master/vendor/css/main.css already contains minified version of materialize.css 

We do not need to re-add. 

Have you tried with adding css dependency. I belive the `chip` class will still work